ecs-logs-collector.sh

```diff
@@ -784,6 +785,20 @@ enable_ecs_agent_debug() {
   fi
 }
 
+# nvidia-smi is a tool available on GPU based AMIs provides detailed
+# information about the GPU present on the VM.
+get_gpu_info() {
+  try "get gpu info"
+
+  if command -v nvidia-smi &>/dev/null; then
+    mkdir -p "$info_system"/gpu
+    nvidia-smi -L > "$info_system"/gpu/gpu-list.txt
+    nvidia-smi -q > "$info_system"/gpu/gpu-info.txt
+
```

Comment:
nit: can you remove this empty line(797)?

Reply:
done